walk-in-interview/src/main/webapp/log-in/business/script.js

```diff
@@ -55,4 +62,50 @@ backButton.addEventListener('click', (_) => {
 const submitButton = document.getElementById('submit');
 submitButton.addEventListener('click', (_) => {
   // TODO(issue/78): send the account & password input to the firebase auth related stuff
+  const account = document.getElementById('account-input').value;
+
+  if (account === '') {
+    setErrorMessage(/* errorMessageElementId= */'error-message',
+        /* msg= */ STRINGS['empty-email'],
+        /* includesDefault= */false);
+    return;
+  }
+
+  const password = document.getElementById('password-input').value;
+
+  setErrorMessage(/* errorMessageElementId= */'error-message',
+      /* msg= */ '',
+      /* includesDefault= */false);
+
+  signIntoBusinessAccount(account, password)
+      .catch(error => {
+        switch(error.code) {
+          case INVALID_EMAIL_ERROR_CODE:
+            setErrorMessage(/* errorMessageElementId= */'error-message',
+                /* msg= */ STRINGS['invalid-email-error'],
+                /* includesDefault= */false);
+            break;
+          case USER_DISABLED_ERROR_CODE:
+            setErrorMessage(/* errorMessageElementId= */'error-message',
+                /* msg= */ STRINGS['user-disabled'],
+                /* includesDefault= */false);
+            break;
+          case USER_NOT_FOUND_ERROR_CODE:
+            setErrorMessage(/* errorMessageElementId= */'error-message',
+                /* msg= */ STRINGS['user-not-found'],
+                /* includesDefault= */false);
+            break;
+          case WRONG_PASSWORD_ERROR_CODE:
+            setErrorMessage(/* errorMessageElementId= */'error-message',
+                /* msg= */ STRINGS['wrong-password'],
+                /* includesDefault= */false);
+            break;
+          default:
+            setErrorMessage(/* errorMessageElementId= */'error-message',
+                /* msg= */ error.message,
```

Comment:
not sure if we should display the error msg to the user - maybe just print it in the console?

Reply:
I will change it to a general error message, and only print the exact cause in the console